core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -175,6 +175,7 @@ object Defaults {
   val SSLClientAuthRequested = "requested"
   val SSLClientAuthNone = "none"
   val SSLClientAuth = SSLClientAuthNone
+  val SSLCypherSuites = ""
```

Comment:
can we make this consistent between Cypher vs Cipher. Since we've Cipher everywhere can we keep that.